charts/localstack/templates/ingress.yaml

```diff
@@ -20,6 +20,7 @@ metadata:
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+    {{- include "localstack.annotations" . | nindent 4 }}
```

Comment:
Same as the podAnnotations above.. since the `ingress.annotations` already exists, it's good enough as it is.

Reply:
Fixed merge function.